sormas-ui/src/main/java/de/symeda/sormas/ui/exposure/ExposuresField.java

```diff
@@ -145,11 +178,42 @@ private void addGeneratedColumns(Table table) {
 			return exposureTypeLabel;
 		});
 
+		table.addGeneratedColumn(COLUMN_EXPOSURE_ROLE, (Table.ColumnGenerator) (source, itemId, columnId) -> {
+			ExposureDto exposure = (ExposureDto) itemId;
+			return exposure.getExposureRole() != null ? exposure.getExposureRole().toString() : "";
+		});
+
 		table.addGeneratedColumn(COLUMN_TYPE_OF_PLACE, (Table.ColumnGenerator) (source, itemId, columnId) -> {
 			ExposureDto exposure = (ExposureDto) itemId;
-			return exposure.getTypeOfPlace() != null
-				? TypeOfPlace.OTHER != exposure.getTypeOfPlace() ? exposure.getTypeOfPlace().toString() : exposure.getTypeOfPlaceDetails()
-				: "";
+			String typeOfPlaceString = "";
```

Comment:
Unnecessary to initialise the String with "" here because it's always overwritten; `String typeOfPlaceString;` is sufficient